website/docs/framework-docs/getting-started/quickstart.md

```diff
@@ -33,12 +33,22 @@ Here, you can give any project name of your choosing.
 
 When you execute this command, you will be asked a set of questions, based on which, the Boyka project will be setup.
 
+:::tip
+Read the questions very carefully and understand it, then reply with your answers
```

Comment:
While answering the questions, make sure you read it carefully and reply accordingly to your requirement.